ansible/roles/software/ckan/inventory/templates/etc/supervisor/conf.d/gunicorn-web-app.conf

```diff
@@ -1,5 +1,5 @@
 [program:gunicorn-web-app]
-command={{ project_source_new_code_path }}/bin/gunicorn --worker-class gevent --paste /etc/ckan/production.ini -b localhost:5000
+command={{ project_source_new_code_path }}/bin/gunicorn --worker-class gevent --paste /etc/ckan/production.ini -b localhost:5000 --proxy-allow-from="127.0.0.1"
```

Comment:
fwiw, I think 127.0.0.1 is the [default](https://docs.gunicorn.org/en/stable/settings.html#proxy-allow-ips)